.github/workflows/tox-linters.yaml

```diff
@@ -0,0 +1,88 @@
+---
+name: 🚨linters
```

Comment:
Please check if this fits in the UI both on the PR main page and on the Checks tab, in the sidebar.

Reply:
Checked. Looks fine.